preview/Dockerfile

```diff
@@ -1,5 +1,16 @@
 FROM buildpack-deps:trusty-scm
 
+# Temporarily work around https://github.com/dotnet/cli/issues/1582 by
+# avoiding the restart_syscall made by LTTng which causes a failed assertion.
+# This workaround specifies that applications should not wait for a sessiond
+# "registration done" command before proceeding to execute the main program.
+#
+# This issue is being addressed in Docker (see
+# https://github.com/docker/docker/issues/20818). When that fix has been
+# included in a release of Docker, this workaround will be removed, and users
+# will be expected to upgrade their Docker installation.
```

Comment:
That makes sense.  I've updated the comment to mostly match your suggestion, but I do like mentioning that it avoids the failed assertion from LTTng.